weave-js/src/core/ops/domain/artifactVersion.ts

```diff
@@ -638,6 +638,7 @@ export const opArtifactVersionRunHistoryRow = makeBasicOp({
   name: 'artifactVersion-historyMetrics',
   argTypes: {
     artifactVersion: TypeHelpers.nullableOneOrMany('artifactVersion' as const),
+    maxKeyLimit: 'number' as const,
```

Comment:
How do I make these params optional?

Reply:
does `maxKeyLimit: TypeHelpers.maybe('number' as const)` work?

Reply:
hmm, just tried this - doesn't seem to work. I can't use the `opArtifactVersionRunHistoryRow` op without specifying a number for the maxKeyLimit argument

Reply:
nevermind, it worked!